Riot/Modules/Room/VoiceMessages/VoiceMessagePlaybackView.swift

```diff
@@ -138,4 +161,31 @@ class VoiceMessagePlaybackView: UIView, NibLoadable, Themable {
     @IBAction private func onPlayButtonTap() {
         delegate?.voiceMessagePlaybackViewDidRequestPlaybackToggle()
     }
+    
+    @IBAction private func tap(gestureRecognizer: UITapGestureRecognizer) {
+        let x = gestureRecognizer.location(in: waveformContainerView).x.clamped(to: 0...waveformContainerView.bounds.width)
+        let progress = x / waveformContainerView.bounds.width
+        let seekPoint = progress == 1 ? 0 : progress
+        delegate?.voiceMessagePlaybackViewDidRequestSeek(to: seekPoint)
+    }
+    
+    @IBAction private func pan(gestureRecognizer: UIPanGestureRecognizer) {
```

Comment:
The waveform doesn't currently update while scrubbing, perhaps we can simplify the pan handling to this:
```
@IBAction private func pan(gestureRecognizer: UIPanGestureRecognizer) {
        switch gestureRecognizer.state {
        case .began, .changed:
            let x = gestureRecognizer.location(in: waveformContainerView).x.clamped(to: 0...waveformContainerView.bounds.width)
            scrubProgress = x / waveformContainerView.bounds.width
            let seekPoint = scrubProgress == 1 ? 0 : scrubProgress ?? 0
            delegate?.voiceMessagePlaybackViewDidRequestSeek(to: seekPoint)
        default:
            scrubProgress = nil
        }
    }
```